src/Sylius/Component/Archetype/Model/Archetype.php

```diff
@@ -29,7 +30,7 @@
  */
 class Archetype extends AbstractTranslatable implements ArchetypeInterface
 {
-    use TimestampableTrait;
+    use CodeAwareTrait, TimestampableTrait;
```

Comment:
Nothing specific about importing traits is mentioned in PSR-2, but if there many to import in the future (hopefully not) it might be worth declaring one per statement, [like when importing classes from other namespaces](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#3-namespace-and-use-declarations).